executor/src/witgen/machines/block_machine.rs

```diff
@@ -151,6 +151,11 @@ impl<'a, T: FieldElement> BlockMachine<'a, T> {
             block_count_runtime: 0,
         })
     }
+
+    #[cfg(test)]
+    pub fn machine_info(&self) -> (MachineParts<'a, T>, usize, usize) {
+        (self.parts.clone(), self.block_size, self.latch_row)
+    }
```

Comment:
This is called only once so not sure it should be a method?

Reply:
It exists to work around the fact that these fields are private!